apps/web/src/app/api/og/_components/tracker.tsx

```diff
@@ -2,6 +2,7 @@ import type { Monitor } from "@openstatus/tinybird";
 import { Tracker as OSTracker, classNames } from "@openstatus/tracker";
 
 import { cn, formatDate } from "@/lib/utils";
+import { nanoid } from "nanoid";
```

Comment:
are you sure about using nanoid?

Reply:
Hey @thibaultleouay ,

Yes, I have done a bit of my research on using a production-grade way to use keys for React elements, and I found this nanoid package, which has over 37 million weekly downloads with a 10.9 KB unpacked size.

If you can suggest any better options, I am happy to work on that.

![Screenshot from 2024-09-07 11-35-41](https://github.com/user-attachments/assets/38f730ef-a061-4f05-9dd9-ab352361b01b)